src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementDialog.tsx

```diff
@@ -41,6 +54,13 @@ const MSG = defineMessages({
     id: 'dashboard.PermissionManagementDialog.selectUser',
     defaultMessage: 'Member',
   },
+  noPermissionFrom: {
+    id:
+      'dashboard.CreatePaymentDialog.CreatePaymentDialogForm.noPermissionFrom',
```

Comment:
I don't think id is correct

Reply:
You are right, must be the result of my copy-pasting :(